utility/src/main/java/org/oppia/android/util/parser/TestGlideImageLoader.kt

```diff
@@ -1,18 +1,48 @@
 package org.oppia.android.util.parser
 
 import android.content.Context
+import android.graphics.Bitmap
 import android.graphics.drawable.Drawable
+import android.graphics.drawable.PictureDrawable
 import org.oppia.android.util.caching.AssetRepository
 import org.oppia.android.util.caching.CacheAssetsLocally
 import javax.inject.Inject
 
-/** A [TestGlideImageLoader] that is used in tests. */
+/** [TestGlideImageLoader] is designed to be used in tests. It uses real [GlideImageLoader]
+ * except for [loadDrawable]. [loadDrawable] function is overridden to work with drawable matchers
+ * in unit tests.
+ */
 class TestGlideImageLoader @Inject constructor(
   context: Context,
   @CacheAssetsLocally private val cacheAssetsLocally: Boolean,
   assetRepository: AssetRepository
-) : GlideImageLoader(context, cacheAssetsLocally, assetRepository) {
+) : ImageLoader {
 
+  private val glideImageLoader by lazy {
+    GlideImageLoader(context, cacheAssetsLocally, assetRepository)
```

Comment:
You should be able to inject this directly (since it has an Inject constructor).

Reply:
Done, injected properly (I think)